Assets/Scripts/Other/ExitChute.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ExitChute : MonoBehaviour
+{
+
+    private LevelController levelController;
+
+    private void Start() {
+        try {
+			levelController = GameObject.Find("The God Monobehaviour").GetComponent<LevelController>();
```

Comment:
Why `GameObject.Find` instead of attaching a reference?

Not a blocker but I'm curious.